low-level/sensors/impl/platforms/realstor/realstor_logical_volume_sensor.py

```diff
@@ -137,18 +158,41 @@ def initialize(self, conf_reader, msgQlist, products):
              self.LOGICAL_VOLUMES_DIR)
 
         # Persistence file location. This file stores faulty Logical Volume data
-        # for logical volume alerts
-        self._faulty_logical_volume_file_path = os.path.join(
+        # and Disk Group data for alerts
+        self._sensor_data_file_path = os.path.join(
```

Comment:
one file has both DG & volume data persisted? why not separate it out. In any case, name of the file should reflect the resources , of which data being stored.

Reply:
Okay, moving disk group data to another file